caf-audit-service/pom.xml

```diff
@@ -134,5 +134,9 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.cafaudit</groupId>
+            <artifactId>caf-audit-elasticclient</artifactId>
+        </dependency>
```

Comment:
This is used to obtain configuration classes and is required at compile time.

Reply:
Is that required?  Shouldn't it just be configured in the normal way that any audit client would be?

Reply:
This is how it was configured already since there is a Jira specifically changing how this is configured I thought it was best to leave that change to happen under that Jira.